common/logisticspipes/gui/GuiGuideBook.java

```diff
@@ -0,0 +1,171 @@
+package logisticspipes.gui;
+
+import java.io.IOException;
+
+import net.minecraft.client.gui.GuiButton;
+import net.minecraft.client.gui.GuiScreen;
+import net.minecraft.client.gui.GuiSlider;
+import net.minecraft.client.gui.inventory.GuiContainer;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.util.ResourceLocation;
+
+import org.lwjgl.Sys;
+
+import logisticspipes.LPConstants;
+import logisticspipes.utils.string.StringUtils;
+
+public class GuiGuideBook extends GuiScreen {
+
+	private static final ResourceLocation GUI_BOOK_TEXTURE = new ResourceLocation(LPConstants.LP_MOD_ID, "textures/gui/guide_book.png");
+
+	private GuiButton nextPageBtn, prevPageBtn, slider;
+	private boolean dragging = false;
+	private int mouseX, mouseY;
+
+	//Book vars
+	private int PAGE = 1;
+	private int PAGE_MAX = 10;
+	private int MAX_LENGTH = 50;
```

Comment:
only constants (static+final) should be title case, make these var names camel case

Reply:
Alright done